### ISSUE TITLE
Latest version low upper cap

### ISSUE THREAD

**Issue by TristanCacqueray** [bitemyapp/bloodhound#279]
Hello, would it be possible to get a new release? The latest one on hackage (0.16.0.0) has these problematic caps:

```
containers >=0.5.0.0 && <0.6,
http-client >=0.4.30 && <0.6,
semigroups >=0.15 && <0.19,
time >=1.4 && <1.9
```

which conflicts with other libraries. It seems like the upper caps are now removed in the current HEAD, so a new release would solve the current conflicts.

**Comment by MichaelXavier** [comment]
I've released version 0.18.0.0 on Hackage. Sorry for the delay. I just don't have much time for this project anymore and have been avoiding it. As I mention in the readme now, please reach out if you're interested in becoming a maintainer.

**Comment by TristanCacqueray** [comment]
That's great, thank you!
